app/src/main/java/com/example/giftly/CThemeScreen.java

```diff
@@ -15,88 +15,100 @@
+            getWindow().getDecorView().setBackgroundColor(savedColor);
+
+            Button dftThemeBtn = findViewById(R.id.defaultTheme);
+            Button glyThemeBtn = findViewById(R.id.giftlyTheme);
+            Button vltThemeBtn = findViewById(R.id.valentinesTheme);
+            Button etrThemeBtn = findViewById(R.id.easterTheme);
+            Button savingit = findViewById(R.id.saveIt);
+
+            savingit.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    Intent intent = new Intent(CThemeScreen.this, HomeScreen.class);
+                    startActivity(intent);
+                }
+            });
+
+            dftThemeBtn.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    int selectedColor = ContextCompat.getColor(CThemeScreen.this, R.color.Default_color);
+                    sharedPreferences.edit().putInt("BackgroundColor", selectedColor).apply();
+                    themeSubject.notifyObservers(selectedColor);
+                }
+            });
+
+            glyThemeBtn.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    int selectedColor = ContextCompat.getColor(CThemeScreen.this, R.color.Giftly_color);
+                    sharedPreferences.edit().putInt("BackgroundColor", selectedColor).apply();
+                    themeSubject.notifyObservers(selectedColor);
+                }
+            });
+            //Valentines
+            vltThemeBtn.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    int selectedColor = ContextCompat.getColor(CThemeScreen.this, R.color.Valentines_color);
+                    sharedPreferences.edit().putInt("BackgroundColor", selectedColor).apply();
+                    themeSubject.notifyObservers(selectedColor);
+                }
+            });
+
+            etrThemeBtn.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    int selectedColor = ContextCompat.getColor(CThemeScreen.this, R.color.Easter_color);
+                    sharedPreferences.edit().putInt("BackgroundColor", selectedColor).apply();
+                    themeSubject.notifyObservers(selectedColor);
+                }
+            });
+        }
+
+        @Override
+        public void onThemeChanged(int newColor) {
+            getWindow().getDecorView().setBackgroundColor(newColor);
+        }
+}
+
+
+    //Back button
+    //ActionBar actionBar = getSupportActionBar();
+    //actionBar.setDisplayHomeAsUpEnabled(true);
+
```

Comment:
Delete the ActionBar methods since the configuration will either be used or deleted.

Reply:
Removed, thank you